### DISCUSSION TITLE
Also include project in `composer show`

### DISCUSSION THREAD

**Discussion by bilogic** [composer/composer#11780] [Q&A / Support] [answered]
```
# composer show
vmitchell85/nova-links                        v2.1             Add custom links to your nova navigation
voku/portable-ascii                           2.0.1            Portable ASCII library - performance optimized (ascii) string functions for php.
webmozart/assert                              1.11.0           Assertions to validate method input/output with nice error messages.
wire-elements/modal                           2.0.9            Laravel Livewire modal component
yassi/nova-nested-form                        v3.0.12          A Laravel Nova package that allows you to create/update/delete nested related fields from a pare...
zbateson/mail-mime-parser                     2.4.0            MIME email message parser
zbateson/mb-wrapper                           1.2.0            Wrapper for mbstring with fallback to iconv for encoding conversion and string manipulation
zbateson/stream-decorators                    1.2.1            PHP psr7 stream decorators for mime message part streams
```

I manage a bunch of packages, so I often use `composer show` in my staging to make sure things are good before final deployment. However, `composer show` does not include details of the project itself. Possible to have an option to add that in too?

**Comment by Seldaek** [comment]
You can use `composer show --self` or `-s` to see the current project. It'll show all details though.

**Reply by bilogic** [reply]
Yea I know, but it shows in a different format.

**Reply by Seldaek** [reply]
Ok, i think i could easily make it include the root package if you explicitly pass both --self --installed (the latter being implicit usually.. But we need to distinguish from simply --self)

**Reply by bilogic** [reply]
Well, as long as there is a parameter to show it. I thank you for being so open and welcoming to suggestions. :)

**Reply by Seldaek** [reply, marked as answer]
See https://github.com/composer/composer/pull/11785 - you can now do `composer show --self --installed` or `composer show --self --locked` to get a list of locked/installed + the root package in the list.
